code/modules/reagents/Chemistry-Reagents/Toxin-Reagents.dm

```diff
@@ -642,6 +642,37 @@
 		playsound(M, "sparks", 50, 1)
 	..()
 
+/datum/reagent/toxin/rotatium //Rotatium. Fucks up your rotation and is hilarious
+	name = "Rotatium"
+	id = "rotatium"
+	description = "A constantly swirling, oddly colourful fluid. Causes the consumer's sense of direction and hand-eye coordination to become wild."
+	reagent_state = LIQUID
+	color = "#FFFF00" //RGB: 255, 255, 0 Bright ass yellow
+	metabolization_rate = 0.6 * REAGENTS_METABOLISM
+	toxpwr = 0
+	var/rotate_timer = 0
+	var/randrot = 1
+	var/hold = 0
+
+/datum/reagent/toxin/rotatium/on_mob_life(mob/living/M)
+	rotate_timer++
+	if(M.reagents.get_reagent_amount("rotatium") < 2)
+		M.client.dir = 1
```

Comment:
@optimumtact can you put this on ```on_mob_delete()``` too?
otherwise people will be stuck with a weird client dir forever in cases where it wasn't removed by camomel, such as admin force removals.